src/engine/core/game_object.rs

```diff
@@ -74,7 +74,7 @@ impl Component {
 pub struct GameObject {
     pub transform: Isometry3<f32>,
     pub scale: Vector3<f32>,
-
+    pub visible: bool,

```

Comment:
Is it possbile to change visible to active ?

Reply:
Yeah no problem. I like unity , but i think godot is even more elegant (while very close) Did you check godot?

Reply:
Yes i idid actually. I love it too but its html5 support is still no so mature (as i test), so that's why i start this project 😜

Reply:
And Look good for me, i would merge it to master branch, Thank you so much.